### ISSUE TITLE
onboarding bot QA

### ISSUE THREAD

**Issue by Keyrxng** [ubq-testing/ask-plugin#2]

**Comment by Keyrxng** [comment]
@ubqbot the rpc-handler sounds awesome what is it?

**Comment by Keyrxng** [comment]
@ubqbot what is conversation rewards I don't understand

**Comment by ghost** [comment]
### Explanation of "Conversation Rewards"

"Conversation Rewards" refers to a plugin or feature that calculates and allocates rewards based on contributions to a conversation (such as a GitHub issue or pull request). Key contributors to the conversation are rewarded based on the meaningfulness, relevance, and nature of their contributions.

#### Formula
The reward formula is as follows:

```
Total Reward = (Σ of (wordCount^exponent × wordValue × relevance) + (score × elementCount)) × multiplier + task.reward
```

#### Breakdown of Components:
- **wordCount**: Number of words in the comment
- **wordValue**: Assigned value per word (e.g., `0.1`)
- **relevance**: How relevant the comment is to the issue or task
- **elementCount**: Counts of specific formatting elements like code blocks, headers, etc.
- **multiplier**: Adjusts reward based on the role (e.g., issue author, pull contributor)
- **task.reward**: A predefined reward for task-related activities (like implementing certain features)
  
#### Configuration
- Configurable by role with different **multipliers** and **rewards** for each contributor type like:
  - Issue author, pull author, collaborator, etc.
- Uses an API-based content evaluator (GPT models) to score content quality.

This helps objectively determine who contributed how much and reward them accordingly in projects or discussions.



<!--
{
  "output": 292,
  "input": 1524,
  "total": 1816
}
--!>

**Comment by Keyrxng** [comment]
@ubqbot is the kernel a yarn repo?

**Comment by Keyrxng** [comment]
@ubqbot anvil is causing me issues in pay.ubq.fi

**Comment by Keyrxng** [comment]
@ubqbot how do I setup the kernel from scratch?

**Comment by Keyrxng** [comment]
@ubqbot what is the devpool?

**Comment by ghost** [comment]
The DevPool is a network that aggregates open bounties from registered partners in the global DevPool network. The directory stores information related to those bounties but does not directly mix them with specific partner bounties.

Here are a few key points:

- **Directory**: Stores aggregated bounties from various registered partners.
- **DevPool**: A global network of registered partners contributing bounties to the directory.
- **Repository**: Hosts bounties for the DevPool Directory but avoids mixing partner-specific bounties. 

In short, the DevPool helps manage and aggregate bounties across a network of partners for easy access and collaboration.

<!--
{
  "output": 135,
  "input": 399,
  "total": 534
}
--!>

**Comment by Keyrxng** [comment]
@ubqbot I need to generate a keypair, which project should I use?

**Comment by Keyrxng** [comment]
@ubqbot How do I use the start stop command?

**Comment by Keyrxng** [comment]
@ubqbot can I use yarn with the kernel?
